setup.py

```diff
@@ -33,7 +33,7 @@ def run_setup():
     gcs = 'google-cloud-storage==1.6.0'
     gcs_oauth2_boto_plugin = 'gcs_oauth2_boto_plugin==1.14'
     apacheLibcloud = 'apache-libcloud==2.2.1'
-    cwltool = 'cwltool==3.0.20201203173111'
+    cwltool = 'cwltool@git+https://github.com/mhpopescu/cwltool@allow_named_pipes'
```

Comment:
Unfortunately we can't take a git dependency into Toil's main branch, because it would prevent a PyPI release. So we can't merge this until the required cwltool is released and we can depend again on a release version.